src/components/ExploreFood.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useContext } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import context from '../context/context';
+import Header from './Header';
+import Footer from './Footer';
+import '../styles/Explore.css';
+import { mealRandomAPI, drinkRandomAPI } from '../services/foodAPI';
+
+const ExploreFood = ({ location: { pathname } }) => {
+  const {
+    setCurrentFood,
+    enableButton,
+    setEnableButton,
+    route,
+    setRoute,
+    idFood,
+    setIdFood,
+  } = useContext(context);
+
+  useEffect(() => {
+    if (pathname === '/explorar/comidas') {
+      setCurrentFood('Explorar Comidas');
+      setEnableButton(false);
+      setRoute('/explorar/comidas/ingredientes');
+      setIdFood('/receitas/comidas/');
+      mealRandomAPI();
+    } else {
+      setCurrentFood('Explorar Bebidas');
+      setEnableButton(true);
+      setRoute('/explorar/bebidas/ingredientes');
+      setIdFood('/receitas/bebidas/');
+      drinkRandomAPI();
+    }
+  }, [pathname]);
+
+  return (
+    <div>
+      <Header />
+      <div className="center-buttons">
+        <Link to={route}>
+          <button data-testid="explore-by-ingredient" className="button">Por ingredientes</button>
+        </Link>
+        <Link to="/explorar/comidas/area">
+          <button
+            data-testid="explore-by-area"
+            className="button"
```

Comment:
Encapsula os CSSs dos buttons, fica mais seguro e evita que dê aqueles bugs de estilização, como tivemos no Header.
```suggestion
            className="Explore_Food_button"
```

Reply:
Feito